src/python/pants/backend/docker/subsystems/docker_options.py

```diff
@@ -56,6 +58,15 @@ class DockerOptions(Subsystem):
 
             A configured registry is marked as default either by setting `default = true`
             or with an alias of `"default"`.
+
+            A `docker_image` may be pushed to a subset of registries using the per registry
+            `skip_push` option rather then the all or nothing toggle of the field option `skip_push`
```

Comment:
Calling it a "field option" is a little confusing. I think leave off the option part.

Reply:
Also maybe use this format? `[docker].skip_push`. although I'm not sure how exactly to handle that with the registry syntax.

Reply:
I can drop the word "option", leaving just "field" (will hold that until there's some more changes). Feel free to tweak the message as you see fit beyond that, if you come up with something you like better ;)